os/arch/arm/include/armv7-a/irq.h

```diff
@@ -199,10 +199,10 @@
 #define REG_SB              REG_R9
 #define REG_SL              REG_R10
 #ifdef CONFIG_ARM_THUMB
-  #define REG_FP            REG_R7
+#define REG_FP            REG_R7
 #else
-  #define REG_FP            REG_R11
-#endif /* CONFIG_ARM_THUMB */
+#define REG_FP            REG_R11
+#endif							/* CONFIG_ARM_THUMB */
```

Comment:
too many tabs. Is this intended for #endif? the same is present in many other places as well.

Reply:
The tabs are allocated as per the script.